packages/commonwealth/.storybook/stories/components/SearchBar.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { CWSearchBar } from "../../../client/scripts/views/components/component_kit/new_designs/cw_search_bar";
+
+const searchbar = {
+  title: "Components/SearchBar",
+  component: CWSearchBar,
+} satisfies Meta<typeof CWSearchBar>;
```

Comment:
curious what is the reason we use `satisfies`? I remember that I removed it some time ago and everything seemed to work fine?

Reply:
Removed. I just added because it was on Storybook docs as reference.